ee/packages/omnichannel-services/src/OmnichannelTranscript.ts

```diff
@@ -275,7 +287,9 @@ export class OmnichannelTranscript extends ServiceClass implements IOmnichannelT
 		}
 		this.currentJobNumber++;
 		try {
-			const room = await LivechatRooms.findOneById(details.rid);
+			const room = await LivechatRooms.findOneById<Pick<IOmnichannelRoom, '_id' | 'v' | 'closedAt' | 'servedBy'>>(details.rid, {
+				projection: { _id: 1, v: 1, closedAt: 1, servedBy: 1 },
+			});
```

Comment:
It seems this is not related to the issue, if so, we can open a new PR for this improvement if necessary.